src/httpsv.c

```diff
@@ -528,6 +528,10 @@ void HTTPSV_GetMethod(cluster_t *cluster, oproxy_t *pend)
 	{
 		HTTPSV_GenerateDemoListing(cluster, pend);
 	}
+	else if (URLCOMPARE(getpath, "/demo_filenames.txt", skiplen))
```

Comment:
Is this URL a good choice? What will happen if I go this URL?
- Do I get a prompt to download a file called 'demo_filenames.txt'?
- Do I get a page with a list of demos?

If the latter (so no download) I think we should not call the page "/demo_filenames.txt", as it is misleading. Just call it "/demo_filenames", without the extension.

Reply:
It depends on your browser settings.